lib/commands/find.js

```diff
@@ -23,12 +23,21 @@ helpers.findUIElementOrElements = async function  (strategy, selector, mult, con
   if (strategy !== "xpath") {
     selector = util.escapeSpecialChars(selector, "'");
   }
+
   if (typeof context === "undefined" || !context) {
     context = '';
   } else if (typeof context === "string") {
     context = util.escapeSpecialChars(context, "'");
   }
 
+  if (strategy === "xpath" && selector === "//*[@scrollable=true]") {
+    return await this.findScrollableElOrEls(mult, context);
+  }
+
+  if (strategy === "xpath" && selector === "//*[@onScreen=true]" && context) {
```

Comment:
use single quotes to enclose a string unless there are another single quotes inside it